src/pages/limyujin/Main/Components/SearchModal.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './SearchModal.scss';
+
+class SearchModal extends React.Component {
+  render() {
+    const { userData, isInputFocused, searchText } = this.props;
+    const filteredData = userData.filter(el => {
+      if (el.author.toLowerCase().includes(searchText)) {
+        return true;
+      }
```

Comment:
```suggestion
      return el.author.toLowerCase().includes(searchText)
```
로 처리 가능해보입니다.
검색기능 filter method 이용해서 잘 구현하신 것 너무 좋네요 굿!

Reply:
아...!! 감사합니다 연욱님 😇 수정 완료했습니다!